airbyte-integrations/connectors/destination-google-sheets/destination_google_sheets/buffer.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Any, List, Mapping
+
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.models import AirbyteStream
+
+
+class WriteBuffer:
+
+    # Default instance of AirbyteLogger
+    logger = AirbyteLogger()
+    # Buffer for input records
+    records_buffer = []
```

Comment:
records_buffer - should be a dict

Reply:
it will simplify code in a lot of places

Reply:
btw, it is possbile that stream_name can be missing in records_buffer?

Reply:
We prepare the buffers at very first step, the names are coming from configured_catalog, so missing stream_name should not be the case.